app/routes/application.js

```diff
@@ -106,6 +106,10 @@ let ApplicationRoute = Route.extend(ApplicationRouteMixin, ModalHelper, SetupUse
   afterModel() {
     set(this.controllerFor('navigation'), 'allowSearch', false);
     $('#apploading').remove();
+    this.get('config.configDB').get('current_user').then((user) => {
+      let language = user.i18n || 'en';
```

Comment:
`_setUserLanguageChoice(language)` saves a new record in the configDB, with the id of the record being the user, but here you are just fetching the current_user, which has an id of 'current_user', not the id of the user.  Does that make sense?